examples/plot_intersectional_bias.py

```diff
@@ -0,0 +1,1327 @@
+#
+# Note that there are some subjective decisions we are making in our fairness assessment. First, we
+# have chosen white men as our reference group, but other groups might be justified here. For
+# instance, if we are interested in racialization related to Black groups specifically, we might
+# choose another racialized group as our reference (e.g., Hispanic men). If we are interested in
+# the impact of gender, we may choose Black women as our reference group.
+#
+# Another decision is related to the metric – we could have also considered some other performance
+# metrics. Selection rate, for example, refers to the proportion of positive predictions between
+# the two groups:
+#
+#   :math:`\text{Selection rate}= \frac{TP + FP}{n}`
+#
+# Because we are primarily interested in misdiagnosis of schizophrenia and not the total number of
+# schizophrenia diagnoses (which indeed may be higher in certain demographic groups), false
+# positive rate is likely a better fit for our task.
+#
+# Critically, there are no readily-agreed upon definitions and understandings of fairness. There
+# are over 70 definitions of fairness, many of which conflict with each other, making it
+# **impossible to simultaneously satisfy all possible metrics for fairness.**
+#
+# Racial bias
+# ~~~~~~~~~~~
+#
+# In order to perform a fairness assessment, there are a few key things we need to do:
+#
+# **1) Which population might be unfairly affected by this model?**
+#
+# Based on the research, we have defined this group to be Black men. In other words, we are
+# interested in a particular intersection of race and sex. In fairness terms, we define race and
+# sex to be **sensitive variables** - features that we want to ensure our model isn't being
+# discriminatory against.
+#
+# We'll start with an assessment purely on the basis of Race as a sensitive variable (How might all
+# Black individuals be affected by an unfair model?) and then add in sex (How might Black men be
+# affected by unfair model?) in order to demonstrate the value of an intersectional approach.
+#
+# **2) What is fairness in this context?**
+#
+# We've also determined a fairness metric, which quantifies the exact nature of the discrimination
+# which groups may face and that we seek to minimize. As we mentioned, various fairness metrics may
+# be relevant, but we are primarily concerned about misdiagnosing individuals with affective
+# disorder as having schizophrenia. We examine false positive rates (i.e., false diagnoses of
+# schizophrenia), with the help of FairLearn.
+#
+# **3) How do we compare populations?**
+#
+# Finally, the performance of a ML model cannot be analyzed in isolation, but rather in comparison
+# between different demographic subgroups. In order to do this, a reference group is required. As
+# we explained, we will use white individuals as a reference group, but this assessment could be
+# conducted with any reference group. It is important to note that this approach can be
```

Comment:
Might be useful to note here that the average can also serve as a reference group, but may also be problematic

Reply:
Good idea!

> As we explained, we will use white individuals as a reference group, but this assessment could be conducted with any reference group, or compared to an overall score across all groups. It is important to note that both approaches can be problematic: there are concerns that using white groups as reference groups centers their perspectives or experiences and represents other groups as other or outliers, or that using overall averages may mask disparities (especially intersectional ones) and ignore important historical and social context. Keeping this in mind, we should carefully consider which group we use as a reference and the implications of this choice.

Reply:
I've committed the suggestion by @yifansmart!